src/components/players/placeholder.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import styles from './placeholder.module.scss';
+
+const DetailsPlaceholder = () => {
+    return (
+        <div className={styles.container}>
+            <div className={styles.placeholderImage}><span>Select a Player for details</span></div>
+            <div className={styles.underline} />
+            <div className={styles.box}>
+                <div className={styles.barContainer}>
+                    <div className={styles.barOne} />
+                    <div className={styles.barTwo} /> 
+                    <div className={styles.barThree} />
+                </div>
+                <div className={styles.circleContainer}>
```

Comment:
Also this one still needs to be done:

https://github.com/kammanz/raptors-app/pull/45#discussion_r393430590